privacyidea/lib/policy.py

```diff
@@ -971,6 +971,9 @@ def get_static_policy_definitions(scope=None):
             ACTION.AUDIT: {
                 'type': 'bool',
                 'desc': _('Allow the user to view his own token history.')},
+            ACTION.USERLIST: {'type': 'bool',
```

Comment:
Oh, you are totally right.
As soon as we define a user policy, the user will not be able to list his own data anymore.
So this needs to be added!
THanks!